power_grid_model_c/power_grid_model/include/power_grid_model/main_model.hpp

```diff
@@ -1191,6 +1200,11 @@ class MainModelImpl<ExtraRetrievableTypes<ExtraRetrievableType...>, ComponentLis
                     y_bus_vec.emplace_back(state_.math_topology[i],
                                            std::make_shared<MathModelParam<sym> const>(std::move(math_params[i])));
                 }
+
+                y_bus_vec.back().set_branch_param_idx(
+                    IdxVector(branch_param_in_seq_map.begin(), branch_param_in_seq_map.end()));
+                y_bus_vec.back().set_shunt_param_idx(
+                    IdxVector(shunt_param_in_seq_map.begin(), shunt_param_in_seq_map.end()));
```

Comment:
curly brace (initalizer-list) constructor is preferred over braces-constructor because of precedence (more-specialized over less-specialized)

```suggestion
                y_bus_vec.back().set_branch_param_idx(
                    IdxVector{branch_param_in_seq_map.begin(), branch_param_in_seq_map.end()});
                y_bus_vec.back().set_shunt_param_idx(
                    IdxVector{shunt_param_in_seq_map.begin(), shunt_param_in_seq_map.end()});
```

Reply:
Resolved in latest commit.